pkg/dataplane/http/context.go

```diff
@@ -16,13 +16,15 @@ import (
 	"strings"
 	"sync/atomic"
 	"time"
+	"io"
 
 	"github.com/v3io/v3io-go/pkg/dataplane"
 	"github.com/v3io/v3io-go/pkg/errors"
-
+	"github.com/v3io/v3io-go/pkg/dataplane/vncapnp"
```

Comment:
See my private `daemon-interface` (in the `pavius` fork) branch and align your schema directory structure to that